README.md

```diff
@@ -10,12 +10,12 @@ Currently, the following subgroups exist which tackle specific larger features
 or areas within the Dev WG. Contact the reponsible leads to join. Your
 contributions are much appreciated!
 
-| Subgroup               | Alias       | Lead            | GitHub                                     | Moderated ML                                |
-| ---------------------- | ----------- | -------------   | ------------------------------------------ | ------------------------------------------- |
-| Namespaces             | NS          | Alex Scheel     | [@cipherboy](https://github.com/cipherboy) | openbao-dev-wg-namespaces@lists.lfedge.org  |
```

Comment:
These have not yet moved fwiw.

Reply:
Looks like I jumped ahead, thanks! I've marked the PR as a draft until we can safely merge it.